src/test.js

```diff
@@ -647,8 +658,11 @@ export function only( testName, callback ) {
 		return;
 	}
 
-	config.queue.length = 0;
-	focused = true;
+	// Only empty the queue in case this test is not part of a skipped module
+	if ( !config.currentModule.skip ) {
```

Comment:
Removed it!

I also removed the `only` test because it will be the only one to run (treated as `skip`) which will ignore my real test that check which tests are skipped & which are not. I hope that's acceptable.